### ISSUE TITLE
Netlify Error When Creating New Task

### ISSUE THREAD

**Issue by carye3** [newjersey/navigator.business.nj.gov#1676]
<img width="1106" alt="image" src="https://user-images.githubusercontent.com/62653834/153447408-95c2d62a-66d0-4399-9110-39c74d554b0b.png">

**Comment by dsueltenfuss** [comment]
@carye3 Are you still receiving this error?
If so, what fields/information did you enter on the form?

**Comment by CeciliaRam** [comment]
@dsueltenfuss preview is no longer blocked by an error. it works!
